#!/usr/bin/env python
PACKAGE = "calibration"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_z",   double_t, 0, "A min height parameter", -0.50, -1.0, 1.0)
gen.add("max_z",   double_t, 0, "A max height parameter", 1.0, -1.0, 1.0)
gen.add("min_ang", double_t, 0, "A min angular parameter", -0.785, -3.14, 3.14)
gen.add("max_ang", double_t, 0, "A max angular parameter", 0.785, -3.14, 3.14)
gen.add("min_dis", double_t, 0, "A min distance parameter", 1.0, 0.0, 5.0)
gen.add("max_dis", double_t, 0, "A max distance parameter", 3.0, 0.0, 5.0)
gen.add("seg_dis", double_t, 0, "A segmentation parameter", 0.05, 0.0, 1.0)

exit(gen.generate(PACKAGE, "calibration", "calibration_lidar"))
